examples/nextjs-whiteboard-advanced/src/index.tsx

```diff
@@ -264,14 +259,14 @@ function Canvas() {
 
       setState({ mode: CanvasMode.Translating, current: point });
     },
-    [canvasState, selection]
+    [canvasState]
```

Comment:
`selection` no longer has to be fed as a dependency. It's now possible to just grab the presence from the `self` argument passed in via the context.